tools/pkf2stl2.cpp

```diff
@@ -0,0 +1,94 @@
+/*!
+  \file pkf2stl2.cpp
+  \ingroup ToolsGroup
+  \brief Read data from a PKF file, produce a tubular mesh for each
+  	curve in the file and write that to an STL file.
+*/
+
+#ifndef DOXYGEN_SHOULD_SKIP_THIS
+
+#include <TubeBundle.h>
+#include <algo_helpers.h>
+
+int main(int argc, char **argv) {
```

Comment:
This could probably have been absorbed as an additional arg to pkf2stl, but this works too.

Mind adding a comment here as to why there is a pkf2stl and a pkf2stl2?

Reply:
Perhaps I will merge them now its clearer how they differ...

Reply:
In this patch or a follow up?

Reply:
Done